cmd/drone-docker/main.go

```diff
@@ -285,6 +290,7 @@ func run(c *cli.Context) error {
 			Disabled:      c.Bool("daemon.off"),
 			IPv6:          c.Bool("daemon.ipv6"),
 			Debug:         c.Bool("daemon.debug"),
+			IPTables:      c.Bool("daemon.iptables"),
```

Comment:
change to `c.BoolT` flag so that it defaults to true

Reply:
done